README.md

```diff
@@ -49,38 +49,58 @@ At this time pya is more suitable for offline rendering than realtime.
 
 `pya` requires `portaudio` to play and record audio. 
 
-1. Disclaimer: As of December 2022, the latest `pyaudio==0.2.12` is not fully available on Conda Forge. This means 
-this methods is currently not suitable, as it will install 0.2.11 instead. This will work on older version of Pya (< 0.5.1). 
+### Using Conda
 
-Anaconda can install non-python packages, so that the easiest way (if applicable) would be to 
+Pyaudio can be installed via [conda](https://docs.conda.io):
 
-    conda install pyaudio
+```
+conda install pyaudio
+```
+
+Disclaimer: Python 3.10+ requires PyAudio 0.2.12 which is not available on Conda as of December 2022. [Conda-forge](https://conda-forge.org/) provides a version only for Linux at the moment. Users of Python 3.10 should for now use other installation options.
 
+### Using Homebrew and PIP (MacOS only)
 
-2. For Mac users, `brew install portaudio`.
-  - For Apple ARM Chip, do the following instead: [Installation on ARM chip](https://stackoverflow.com/a/73166852/4930109)
-      - `brew install portaudio`
-      - `brew --prefix poraudio`
-      - Create .pydistutils.cfg in your home directory, `~/.pydistutils.cfg`, add:
+
+```
+brew install portaudio
+```
+
+For Apple ARM Chip, do the following instead: [Installation on ARM chip](https://stackoverflow.com/a/73166852/4930109)
+  - `brew install portaudio`
+  - `brew --prefix poraudio`
+  - Create .pydistutils.cfg in your home directory, `~/.pydistutils.cfg`, add:
 
 ```
 [build_ext]
 include_dirs=<PATH FROM STEP 3>/include/
 library_dirs=<PATH FROM STEP 3>/lib/
 ```
```

Comment:
Combining steps after brew install.

- Create a .pydistutils.cfg in your home directory:

```
echo "[build_ext]
include_dirs=$(brew --prefix portaudio)/include/
library_dirs=$(brew --prefix portaudio)/lib/" > ~/.pydistutils.cfg
```

I'd personall skip this step and rather pass aguments to pip. Or via CFLAGS. In my case:

```
CFLAGS="-I/opt/homebrew/include -L/opt/homebrew/lib" pip install pyaudio
```

was enough.

Reply:
There should be at least a remark, that .pydistutils.cfg can be deleted after installation.

Reply:
Fixed typo, reword a little more. And add CFLAGS as second option